src/datamigration/azext_datamigration/manual/helper.py

```diff
@@ -278,11 +319,10 @@ def check_and_download_loginMigration_console_app(downloads_folder):
             if user_input == "yes":
                 print("Updating to the new version...")
                 break
-            elif user_input == "no":
+            if user_input == "no":
```

Comment:
Any reason as to why are if removing the `elif`?

Reply:
This is due to pylint checks which is part of Az CLI extension CI/CD pipeline. Below is the error we were getting

![image](https://github.com/user-attachments/assets/bec2f4d2-20ab-4ebd-b16e-99a6fe692167)